app/support/auto_logout.rb

```diff
@@ -13,11 +15,16 @@ def perform
 
   def order_details
     OrderDetail.purchased_active_reservations.
-      where("reservations.actual_end_at IS NULL AND reserve_end_at < ?", Time.zone.now - 1.hour).
+      where("reservations.actual_end_at IS NULL AND reserve_end_at < ?", Time.zone.now).
```

Comment:
Should this use the `Reservation.in_progress` with an additional `where('reserve_end_at < ?)`?